src/content/nft/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans. But digital art is only one way to use NFTs. Really they can be used to represent ownership of any unique asset, like a deed for an item in the digital or physical realm.
+
+_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car might be managed with NFTs._
+
+## What's an NFT?
+
+NFTs are tokens that we can use to represent a record of ownership. They let us tokenise unique items like art, collectibles, even real estate. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.
+
+NFT stands for non-fungible token. Non-fungible is an economic term that you could use to describe things like your furniture, a song file, or your computer. These things are not exchangeable for other items because they have unique properties.
+
+Fungibile items on the other hand can be exchanged as their value defines them rather than their unique properties. For example ETH is fungible because 1 ETH is exchangeable for another ETH.
+
+### NFT examples
+
+THe NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:
+
+- A unique digital artwork.
+- #23/#100 of a limited edition run of designer sneakers.
+- An in-game item.
+- An essay.
+- A digital collectible like a trading card.
+- A domain name...
+
```

Comment:
Could also represent right to access something like an event, like a digital ticket

Reply:
great idea. Can you think of any projects that have done this?

Reply:
Not a lot of live events lately, but I have seen interviews with Mark Cuban who owns the Dallas Mavericks, and has commented on how he wants to use NFT's for tickets to get royalties with any re-sale. Don't think they've done it yet though. 

Another semi-example would be POAPs being used to gain special access or reduced prices at DevCon.